clusterloader2/testing/l4ilb/config.yaml

```diff
@@ -0,0 +1,120 @@
+#Constants
+{{$LARGE_BACKEND_SIZE := 300}}
+{{$MEDIUM_BACKEND_SIZE := 150}}
+{{$SMALL_BACKEND_SIZE := 10}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$ilbQPS := 500}}
+{{$namespaces := 1}}
+#Test
+name: ilbload
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+- name: ILBConstantQPS
+  qpsLoad:
+    averageQps: {{$ilbQPS}}
+steps:
+# Create ILBs
+- measurements:
+  - Identifier: ServiceCreationLatency
+    Method: ServiceCreationLatency
+    Params:
+      action: start
+  - Identifier: WaitForRunningDeployments
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: apps/v1
+      kind: Deployment
+      labelSelector: name = ilbload
```

Comment:
Pods from depl.yaml don't have a matching label.

Reply:
I do have this in dep.yaml:
  selector:
    matchLabels:
      name: {{.Name}}

and test name is ilbload. Can't the selector be used as "name = ilbload"?

Reply:
In the object yaml, {{.Name}} translates to basename-index, which is an object name - it's the cluster loader convention). It will be different for every object instance.
Moreover, labelSelector should match label from deployment.